full-service/src/config.rs

```diff
@@ -89,6 +89,10 @@ pub struct APIConfig {
     /// T3 Server to connect to and the api key to use for authorization.
     #[clap(flatten)]
     pub t3_sync_config: T3Config,
+
+    /// Webhook configuration.
+    #[clap(long, env = "ACCOUNT_SYNC_WEBHOOK_URL")]
```

Comment:
All our of our env variables start with MC_ currently, so adding that... Also this is a webhook that will be fired when there are new deposits, so suggesting alternate name for the argument.
```suggestion
    #[clap(long, env = "MC_DEPOSITS_WEBHOOK_URL")]
```